### PR TITLE
Add HTML pricing template

### LINKED ISSUE #7: Add pricing content

### PR DESCRIPTION
**Pull request by oluwasegunatanda**
resolve #7

### CODE REVIEW

File: html/app.html [hunk 1]
```diff
@@ -20,7 +20,110 @@
...
+              <td><img src="../img/pricing/tick.png" alt="Tick Box" /></td>
+              <td><img src="../img/pricing/tick.png" alt="Tick Box" /></td>
+            </tr>
+          </table>
+
+          <ul>
+            <li>Price</li>
+            <li><img src="../img/pricing/dollar1.png" alt="" /></li>
+            <li><img src="../img/pricing/dollar2.png" alt="" /></li>
+            <li><img src="../img/pricing/dollar3.png" alt="" /></li>
```
Comment (d-code-h) [inline html/app.html:93]: No alternative message for images.

File: html/app.html [hunk 2]
```diff
@@ -20,7 +20,110 @@
...
+          </ul>
+
+          <button>Get Started</button>
+          <button>Get Started</button>
+          <button>Get Started</button>
+        </section>
+
+        <section id="hello">
+          <h3>Say hello</h3>
+          <img src="../img/pricing/Hello.png" alt="" />
```
Comment (d-code-h) [inline html/app.html:103]: The image should be beside a container that contains all hello contents.

File: html/app.html [hunk 3]
```diff
@@ -20,7 +20,110 @@
...
+            <label for="email">EMAIL</label>
+            <input type="email" name="email" placeholder="Your email address" />
+            <label for="message">MESSAGE</label>
+            <textarea
+              name="message"
+              placeholder="Start typing here"
+              cols="30"
+              rows="10"
+            ></textarea>
+            <button>
```
Comment (d-code-h) [inline html/app.html:120]: button type issue

File: html/app.html [hunk 4]
```diff
@@ -20,7 +20,110 @@
...
+              <td><img src="../img/pricing/tick.png" alt="Tick Box" /></td>
+            </tr>
+          </table>
+
+          <ul>
+            <li>Price</li>
+            <li><img src="../img/pricing/dollar1.png" alt="" /></li>
+            <li><img src="../img/pricing/dollar2.png" alt="" /></li>
+            <li><img src="../img/pricing/dollar3.png" alt="" /></li>
+          </ul>
```
Comment (d-code-h) [inline html/app.html:94]: These can be a tfooter in the table rather than an unordered list here.

File: html/app.html [hunk 5]
```diff
@@ -20,7 +20,110 @@
...
+          <img src="../img/pricing/Hello.png" alt="" />
+          <form action="#" method="post">
+            <label for="name">NAME</label>
+            <input type="text" placeholder="Your name" />
+            <label for="subjet">SUBJECT</label>
+            <input type="text" placeholder="Choose subjet" />
+            <label for="company">COMPANY</label>
+            <input type="text" placeholder="Text here" />
+            <label for="email">EMAIL</label>
+            <input type="email" name="email" placeholder="Your email address" />
```
Comment (yunushabeeb) [inline html/app.html]: Inputs does not have name attribute. Also this should be as tfoot in the table above.

File: html/app.html [hunk 6]
```diff
@@ -20,7 +20,110 @@
...
+          <ul>
+            <li>Price</li>
+            <li><img src="../img/pricing/dollar1.png" alt="" /></li>
+            <li><img src="../img/pricing/dollar2.png" alt="" /></li>
+            <li><img src="../img/pricing/dollar3.png" alt="" /></li>
+          </ul>
+
+          <button>Get Started</button>
+          <button>Get Started</button>
+          <button>Get Started</button>
```
Comment (d-code-h) [inline html/app.html:98]: Button does not have a type attribute. Something like <button type="button"> Get Started </button>. Also this should be in the tfoot too.

File: html/app.html (hunk 2 above)
Comment (yunushabeeb) [inline html/app.html]: No alternative message for image.

File: html/app.html [hunk 7]
```diff
@@ -20,7 +20,110 @@
...
+          <form action="#" method="post">
+            <label for="name">NAME</label>
+            <input type="text" placeholder="Your name" />
+            <label for="subjet">SUBJECT</label>
+            <input type="text" placeholder="Choose subjet" />
+            <label for="company">COMPANY</label>
+            <input type="text" placeholder="Text here" />
+            <label for="email">EMAIL</label>
+            <input type="email" name="email" placeholder="Your email address" />
+            <label for="message">MESSAGE</label>
```
Comment (yunushabeeb) [inline html/app.html]: No name attributes for inputs

File: html/app.html (hunk 3 above)
Comment (yunushabeeb) [inline html/app.html:119]: No type attribute

**Review by yunushabeeb — COMMENTED** [review]
Check out the app.html file for comments.

**Comment by yunushabeeb** [comment]
All comments still yet to be fix.

**Pull request closed without merging**